content/response-to-reviewers.md

```diff
@@ -11,29 +11,48 @@ Add responses to this document.
+Please see discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/41>.
+
+> Additionally, the two main conclusions of the paper are so obvious that is difficult to understand the need of the paper. First, conferences and societies in the field are doing an effort to maintain a healthy gender balance, even if clearly far from perfect the interpretation of the results seems to show a positive trend. Instead of analysing the results he authors go into a very long discussion of the causes and consequences,  a discussion that is potentially more appropriate for an opinion paper than for a scientific paper in the conference. The second aspect, is the geographical bias. Yes, it is obvious that most of the speakers are white, the important question to understand the origin of the bias is if there are other more influential authors in the literature that the ones that have been selected in these conferences . The paper does not provide the necessary data to assess is this is the case. To assume that the ideal situation will be to have a number of invitations/honors  proportional to the number of papers by region does not make any sense from a scientific or from the conference organisation point of view.
 
-> Additionally, the two main conclusions of the paper are so obvious that is difficult to understand the need of the paper. First, conferences and societies in the  field are doing an effort to maintain a healthy gender balance, even if clearly  far from perfect the interpretation of the results seems to show a positive trend. Instead of analysing the results he authors go into a very long discussion of the causes and consequences,  a discussion that is potentially more appropriate for an opinion paper than for a scientific paper in the conference. The second aspect, is the geographical bias. Yes, it is obvious that most of the speakers are white, the important question to understand the origin of the bias is if there are other more influential authors in the literature that the ones that have been selected in these conferences . The paper does not provide the necessary data to assess is this is the case. To assume that the ideal situation will be to have a number of invitatio!
->  ns/honors  proportional to the number of papers by region does not make any sense from a scientific or from the conference organisation point of view.
+We agree that the lack of diversity has been readily apparent for some time.
+While the paper's corresponding author curated the list of honorees in 2018 and shared the issues with ISCB board of directors members, we did not observe any changes for 2019 despite these private attempts to influence diversity.
+Since our paper's release, ISCB has released the 2020 set of Fellows: https://www.iscb.org/iscb-fellows, which is 2x larger than most years and includes two scientists with affiliations in Asia including the first ever with an affiliation in China.
```

Comment:
```suggestion
Since our paper's release, ISCB has released the [2020 set of Fellows](https://www.iscb.org/iscb-fellows), which is two times larger than most years and includes two scientists with affiliations in Asia including the first ever with an affiliation in China.
```